lib/ruby_libstorj/env.rb

```diff
@@ -13,8 +13,12 @@ class Env
     end
 
     def initialize(*options)
-      @pointer = ::LibStorj::Ext::Storj.method(:init_env).call(*options)
-      @pointer[:loop] = ::Libuv::Ext.default_loop
+      @storj_env = ::LibStorj::Ext::Storj.method(:init_env).call(*options)
+      @storj_env[:loop] = ::Libuv::Ext.default_loop
```

Comment:
alright - so just out of curiosity i checked and i think ruby-libuv doesn't support changing the internal loop by out of the box... we could tweak it in our binding library but unless there's something specific that you think we should avoid by doing that, I'd say that setting this way is ideal.

could you plz merge if you're good with it :D